### DISCUSSION TITLE
PyPal複習機器人 沒有反應

### DISCUSSION THREAD

**Discussion by chupochun** [1101-GS4719-C/Python-Community-GS4719-C#556] [Q&A]
請問有人知道，為甚麼我的PyPal沒有反應嗎，還是我使用方式錯誤
霸托救救我的通識，我不想被當ㄚㄚㄚㄚㄚ........

![螢幕擷取畫面 2022-04-20 005539](https://user-images.githubusercontent.com/99793902/164055974-bcab2924-f292-4c9f-8b9f-59017a6d8322.png)
![螢幕擷取畫面 2022-04-20 005553](https://user-images.githubusercontent.com/99793902/164056009-9dfca8e8-6e8a-4c43-94e6-9c0df4dcc398.png)

**Comment by chia-kai-chang** [comment]
同學到Discord的玉山會議室~~~~~~ 我等你~

**Reply by chupochun** [reply]
老師抱歉，我剛剛去詢問同學，沒看到留言

**Comment by chia-kai-chang** [comment]
謝謝同學 @chupochun  的提醒，今天稍早有一個地方程式出錯了~

剛剛已經修復~

@badguy361 助教登記一下， @chupochun 學期成績加0.1分

**Comment by chupochun** [comment]
謝謝老師 @benctw 我剛剛借用同學的，稍微複習過了
